docs/cell_census_schema_0.2.0.md

```diff
@@ -718,7 +720,7 @@ The following columns MUST be included:
 </tbody>
 </table>
 
-#### Feature dataset presence matrix – `census_obj[“census_data”][organism].ms[“RNA”].varp[“dataset_presence”]` – `SOMASparseNDArray`
+#### Feature dataset presence matrix – `census_obj[“census_data”][organism].ms[“RNA”][“feature_dataset_presence_matrix”]` – `SOMASparseNDArray`
```

Comment:
It's _possible_ that someone cuts and pastes this to use in actual Python code. Of course _I_ never did that, but I happen to know that the fancy quotes need to be edited after pasting. :). Could just use ASCII double-quote here.

Reply:
I changed all quotes to ASCII